dataRetriever/interface.go

```diff
@@ -186,7 +186,6 @@ type ShardedDataCacherNotifier interface {
 	RemoveSetOfDataFromPool(keys [][]byte, cacheId string)
 	RemoveDataFromAllShards(key []byte)
 	MergeShardStores(sourceCacheID, destCacheID string)
```

Comment:
this function was there to be used when shards are split.

Reply:
@sasurobert, I removed the function according to previous review of @AdoAdoAdo.